### PR TITLE
Update to Extension lib 

### PR DESCRIPTION
**Pull request by it-hms**
Waiting for https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/pull/104

### CODE REVIEW

**Comment by it-hms** [comment]
Build fails because ext lib v1.16.2 is not yet built by automation. Will re-run actions

File: web-docs/docs/02-CHANGELOG.mdx [hunk 1]
```diff
@@ -5,6 +5,11 @@ sidebar_label: Change Log
 toc_max_heading_level: 2
 ---
 
+## Version 1.0.4
+### Bug Fixes
+- Update Extension library to v1.16.2
+- Fixes issue where local timezone offset is only calculated at startup
```
Comment (alexjhawk) [inline web-docs/docs/02-CHANGELOG.mdx:11]: Not the end of the world, but `Update Extension library to v1.16.2` on its own isn't really a bug fix. 

I think that could be a sub-bullet underneath the 'Fixes issue ...' line (or the other way around)

Comment (TomKimsey) [reply web-docs/docs/02-CHANGELOG.mdx:11]: +1 on the sub bullet

Comment (it-hms) [reply web-docs/docs/02-CHANGELOG.mdx]: Changed to sub bullet

**Review by TomKimsey — APPROVED** [review]

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**